presentation/src/main/java/com/knocklock/presentation/lockscreen/StartApplicationReceiver.kt

```diff
@@ -0,0 +1,35 @@
+package com.knocklock.presentation.lockscreen
+
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+
+/**
+ * @Created by 김현국 2022/12/13
+ * @Time 4:00 PM
+ */
+class StartApplicationReceiver : BroadcastReceiver() {
+
+    override fun onReceive(context: Context?, intent: Intent?) {
+        when (intent?.action) {
+            Intent.ACTION_SCREEN_OFF -> {
+                startLockScreen(context)
+            }
+            Intent.ACTION_BOOT_COMPLETED -> {
+                startLockScreen(context)
+            }
+        }
+    }
+
+    private fun startLockScreen(context: Context?) {
+        if (context != null) {
+            try {
+                val intent = Intent(context, LockScreenActivity::class.java)
+                intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
+                context.startActivity(intent)
+            } catch (e: Exception) {
+                println(e.stackTraceToString())
```

Comment:
```suggestion
                Log.e("lockScreen", e.stackTraceToString())
```

prinln보다는 Log를 사용하는게 좋은 것 같아요! cchttps://source.android.com/docs/setup/contribute/code-style#use-todo-comments

- debug, error, info 등 프로젝트 차원에서 로그를 관리, 필터링 하기 좋아요

Reply:
@jaeryo2357 [19de7b0](https://github.com/Knock-Lock/Knock-Lock/pull/61/commits/19de7b0375abf3e9654310379d5cd9fda829d095) 반영했습니당